remote_cypress_manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "schema-version": "1.0",
+  "build": {
+    "name": "OpenSearch Dashboards Functional Test",
+    "version": "2.12.0"
```

Comment:
On main the version should be 3.0.0 I think.

Reply:
That's right. Updated.